docs/helpers/WebDriverIO.md

```diff
@@ -3,23 +3,25 @@
 WebDriverIO helper which wraps [webdriverio](http://webdriver.io/) library to
 manipulate browser using Selenium WebDriver or PhantomJS.
 
-#### Selenium Installation
+## Backends
+
+### Selenium Installation
```

Comment:
this text is generated from corresponding helper. Please update the text inside a docblock in WebDriverIO.js